<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileTransDialog</class>
 <widget class="QDialog" name="FileTransDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>发送文件</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>From:</string>
     </property>
     <property name="buddy">
      <cstring>fromEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="fromEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>To:</string>
     </property>
     <property name="buddy">
      <cstring>toEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="toEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>文件：</string>
     </property>
     <property name="buddy">
      <cstring>fileEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="fileEdit"/>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="browseButton">
     <property name="text">
      <string>浏览</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>大小：</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="sizeLabel">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="maximumSize">
      <size>
       <width>313</width>
       <height>124</height>
      </size>
     </property>
     <property name="title">
      <string>描述：</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="descriptionEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>准备发送文件...</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="sendButton">
     <property name="text">
      <string>发送</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="closeButton">
     <property name="text">
      <string>关闭</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
